tool/grind.dart

```diff
@@ -58,7 +58,7 @@ void outlineIcons() async {
 void _createPng(
   File sourceSvg,
   String targetName, {
-  @required int size,
+  @required int? size,
```

Comment:
Ah, we've bumped to null safe?

We can likely switch from using `@required` to the `required` keyword then as well.